packages/terra-navigation-side-menu/src/_MenuItem.jsx

```diff
@@ -99,28 +98,29 @@ class MenuItem extends React.Component {
       customProps.className,
     ]);
 
+    /* eslint-disable jsx-a11y/no-static-element-interactions, jsx-a11y/no-noninteractive-tabindex */
     return (
-      <List.Item
+      <li
         className={cx('list-item')}
-        content={(
-          /* eslint-disable jsx-a11y/no-static-element-interactions, jsx-a11y/no-noninteractive-tabindex */
-          <div
-            {...customProps}
-            tabIndex="0"
-            className={itemClassNames}
-            onKeyDown={this.handleKeyDown}
-            onKeyUp={this.handleKeyUp}
-            onBlur={this.handleOnBlur}
-          >
-            <div className={cx('title')}>
-              {text}
-            </div>
-            {hasChevron && <span className={cx('chevron')}><ChevronRight /></span>}
+        aria-selected="false"
```

Comment:
shouldn't this value toggle with the `isSelected` prop value?

Reply:
Good catch, looks like this was a bug in the previous code as well.

Reply:
Updated.